vendor/api/v1/authorizenet/views.py

```diff
@@ -84,28 +72,43 @@ def subscription_save_transaction(site, json_data, transaction_detail):
     try:
         payment = subscription.payments.get(transaction=None, status=PurchaseStatus.QUEUED)
         payment.transaction = transaction_id
-        payment.status = PurchaseStatus.CAPTURED
-        payment.result = {}
-        payment.result['raw'] = json_data
+        payment.submitted_date = submitted_datetime
+        payment.result[timezone.now().strftime("%Y-%m-%d_%H:%M:%S")] = payment_info
```

Comment:
Not certain how this is being, but will their ever need to be a need to access `payment.result`? If so, I see this key name being an issue. If not, then its fine

Reply:
It is just logging an when the event happened and with what info for any references we might need.